### DISCUSSION TITLE
Feature: Instagram embed support

### DISCUSSION THREAD

**Discussion by tdresser** [aidenlx/media-extended#286] [Website]
I've got a lot of links to Instagram videos in my workspace.

It would be awesome to be able to embed Instagram videos.

If there's appetite, I might craft a PR at some point.

**Comment by aidenlx** [comment]
It should be easy if you are trying to embed the iframe automatically, but using advanced features like timestamp will require support from Instagram iframe API. I'm not so familiar with it, but it seems to be less promising as I found [this issue](https://github.com/sampotts/plyr/issues/450). 

And of course, feel free to open PR at any point!
